syntax = "proto2";
option optimize_for = LITE_RUNTIME;
option java_package = "ProtoMessages";
option java_outer_classname = "RoomShootDirectionBase";

enum RoomShootDirection {
    rsdLEFT = 0;
    rsdRIGHT = 1;
}